### PR TITLE
opendbx 1.4.6

### PR DESCRIPTION
**Pull request by Habbie**

### CODE REVIEW

File: Library/Formula/opendbx.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class Opendbx < Formula
+  desc "Extremely lightweight but extensible database access library in C"
+  homepage "http://www.linuxnetworks.de/doc/index.php/OpenDBX"
+  url "http://linuxnetworks.de/opendbx/download/opendbx-1.4.6.tar.gz"
+  sha256 "2246a03812c7d90f10194ad01c2213a7646e383000a800277c6fb8d2bf81497c"
+
+  depends_on "sqlite"
+
+  def install
+    inreplace "utils/Makefile.in", "$(LIBSUFFIX)", ".dylib"
```
Comment (mistydemeo) [inline Library/Formula/opendbx.rb]: Can you report this to the developers?

Comment (Habbie) [inline Library/Formula/opendbx.rb]: Yes, I will try. Is this a dealbreaker for homebrew inclusion right now?

Comment (Habbie) [inline Library/Formula/opendbx.rb]: Reported at http://bugs.linuxnetworks.de/index.php?do=details&id=40

Comment (mistydemeo) [inline Library/Formula/opendbx.rb]: Nope, just helpful to have it reported so we don't need to do this forever! Thanks.

**Comment by mistydemeo** [comment]
Merged with some tweaks; thank you!

**Comment by Habbie** [comment]
:+1:

**Pull request closed without merging**